blaze/expr/core.py

```diff
@@ -499,7 +499,7 @@ def common_subexpression(expr, *exprs):
     # no common leaves therefore no common subexpression
     if not leaves:
         raise ValueError(
-            'No common leaves found in expressions %s' % list(exprs)
+            'No common leaves found in expressions %s' % list(map(str, exprs))
```

Comment:
I'm not sure this is the correct fix since in general it doesn't give as much info as the repr does.

I see however that blaze is moving towards a repr which doesn't compute and therefore won't be prone to throwing an exception so this won't be an issue in future.